release.gradle

```diff
@@ -1,5 +1,5 @@
 ext.versionAndRelease = {
-    def version = System.getenv('TAG_NAME') ?: '1.0.0'
+    def version = System.getenv('TAG_NAME') ?: '1.1.3'
```

Comment:
Version `1.0.0` was missing the `SprigDestination`, so this needed to be updated to get the example app to build as part of the PR GitHub action